install-omb.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
```

Comment:
I don't think this is the proper commit to do this change, can you add it into the commit: "Add install-omb.sh"